docs/siem/detections/api/rules-api-create.asciidoc

```diff
@@ -1,13 +1,16 @@
 [[rules-api-create]]
 === Create rule
 
-Creates a new signal detection rule.
+Creates a new detection rule.
 
-You can create two types of rules:
+You can create these types of rules:
 
-* Query-based rules, which search the defined indices and creates a signal when
+* Query rules: Searches the defined indices and creates an alert when
 a document matches the rule's query.
-* {ml-cap} rules, which create a signal when a {ml} job discovers an anomaly above the defined threshold (see <<machine-learning>>).
+* Threshold rules: Searches the defined indices and creates an alert when the
+number of times the specified event field is matched during a single execution
+meets the threshold. 
```

Comment:
Not a professional but could we switch the phrase "meets the threshold" and "matched during a single execution"? The resulting description could be:


"Searches the defined indices and creates an alert when the number of times the specified event meets the threshold during a single execution."?

Reply:
++, It may be worth working in that results are grouped/bucketed/aggregated by the specified event field, and that a detection alert will be created for each bucket that surpasses the threshold.

Reply:
Thanks - @spong @dhurley14 
I tried explaining this more generally so users who don't care/know about ES aggregations understand. Let me know if you have more suggestions or corrections. Thanks